cluster/config-defaults.yaml

```diff
@@ -279,6 +279,9 @@ skipper_oauth2_ui_login_tokeninfo_keys: ""
 # skipper_cluster_scaling_schedules: "schedule1=3,schedule2=5"
 skipper_cluster_scaling_schedules: ""
 
+# Skipper Ingress/RouteGroup backend traffic split algorithm: traffic-predicate or traffic-segment-predicate
+skipper_backend_traffic_algorithm: "traffic-predicate"
```

Comment:
skipper_ -> skipper_ingress

We have too many config items for different parts and please add this more to the core and not here, because `skipper_cluster_scaling_schedules` is some special hpa setting, so all flags to skipper close to each other.

Reply:
I can change that but we are far away from consistency:
```
grep -E ^skipper_ cluster/config-defaults.yaml | grep skipper_ingress | wc -l
33
grep -E ^skipper_ cluster/config-defaults.yaml | grep -v skipper_ingress | wc -l
66
```

Reply:
Changed to `skipper_ingress_backend_traffic_algorithm`